### ISSUE TITLE
Lyrics display error

### ISSUE THREAD

**Issue by EdisonJwa** [JunyuKuang/Rhythm#3]
- **The problem I faced:**

If i'm using Apple music with a non-Chinese district account listening to some Chinese music , the **music name** will be **translated to the account area based language**. And sometimes this will cause that **some music's lyric could not displaying correctly**.
For Example, I was listened to a music called “`不再联系`” by the artist named “`夏天Alex`” but cause there some problems maybe, there was only shows "`Alex`" on Apple Music with US District (Maybe also  for other district).

**NOT ONLY** this song but also other songs will face this problem.
- **The way to resolve it (my opinon)**

So in my opinion if there was a **search bar**(default hidden, scroll down to display) which wil let the users to **search for the lyrics**, would be better to **resolve this problem**.

**The BUG Example is Below:**

![](https://i.loli.net/2019/05/18/5ce00cbc1553963894.jpg)

**Comment by JunyuKuang** [comment]
I agree that providing a search bar for manually lyrics searching can mitigate this issue. But it may cause overburdening at the same time. Still need a better way to solve this problem.

**Comment by EdisonJwa** [comment]
yes

**Comment by JunyuKuang** [comment]
A potential approach is to fetch Chinese version of music info from China mainland’s storefront via [Apple Music API](https://developer.apple.com/documentation/applemusicapi), and then fetch lyrics with those Chinese info.

**Comment by EdisonJwa** [comment]
> A potential approach is to fetch Chinese version of music info from China mainland’s storefront via [Apple Music API](https://developer.apple.com/documentation/applemusicapi), and then fetch lyrics with those Chinese info.

It sounds great but the fact is some songs are not available in China (according to the different area, the songs provided by Apple are different) not sure will these song have chinese info or not
